docs/source/timers.rst

```diff
@@ -17,6 +17,11 @@ this, let us modify the :ref:`push_pull` example a bit and make use of the
 Note that if an action takes longer to run than the time available before the
 next execution, the timer will simply fall behind.
 
+.. note:: To be able to run the ``each()`` method, the agent must be already
+   running, which means you cannot call it from ``on_init()``. Instead, you can
+   use ``configure()``, which will be executed when the agent is already
+   running, right before starting the main loop.
```

Comment:
I would rather say: "To be able to start timers on agent initialization, the agent must be[...]".